{"uuid": "2f0b4ab0-1f18-4556-bd51-ba8da9e12612", "children": ["6d4fec48-57b3-4ce9-b31c-0d3f6b6a119a"], "befores": [{"name": "screenshot_on_failure", "status": "passed", "start": 1597851413141, "stop": 1597851413141}], "afters": [{"name": "screenshot_on_failure::fin", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_setup'\n", "trace": "  File \"c:\\users\\m.kononova\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\m.kononova\\environments\\flexite_autotests\\conftest.py\", line 44, in fin\n    if request.node.rep_setup.failed:\n"}, "start": 1597851417305, "stop": 1597851417519}], "start": 1597851413141, "stop": 1597851417519}